packages/create-block-interactive-template/plugin-templates/$slug.php.mustache

```diff
@@ -42,15 +42,10 @@ if ( ! defined( 'ABSPATH' ) ) {
  * @see https://developer.wordpress.org/reference/functions/register_block_type/
  */
 function {{namespaceSnakeCase}}_{{slugSnakeCase}}_block_init() {
-	register_block_type( __DIR__ . '/build' );
-
-	if (function_exists('gutenberg_register_module')) {
-		gutenberg_register_module(
-			'{{namespace}}-view',
-			plugin_dir_url( __FILE__ ) . 'src/view.js',
-			array( '@wordpress/interactivity' ),
-			'{{version}}'
-		);
+	if ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '17.5', '>=' ) ) {
+		register_block_type_from_metadata( __DIR__ . '/build' );
+		return;
 	}
```

Comment:
What's this for?

Reply:
This was intended to work with the following block, we should either drop the conditional here, or revert the commit that removes the manual module registration.

The metadata registration won't work prior to Gutenberg 17.5.

Reply:
Ok, thank you.

This template already requires GB 17.2, so why not update it to require 17.5. After all, is just a way [to get started with the Interactivity API](https://github.com/WordPress/gutenberg/blob/trunk/packages/interactivity/docs/1-getting-started.md), so requiring the latest version of Gutenberg makes sense.

I've pushed a commit with the changes but please review it yourself as well 🙂

Reply:
That seems fine to me 👍 although we will have a window where we require an unreleased Gutenberg version. Maybe we can dist-tag the previous version of this package as latest until 17.5 is released.

Reply:
It's one week only, I think it's fine.